linkis-commons/linkis-storage/src/main/java/org/apache/linkis/storage/source/FileSource.java

```diff
@@ -80,6 +80,14 @@ static boolean isResultSet(FsPath fsPath) {
     return isResultSet(fsPath.getPath());
   }
 
+  int getTotalCount();
```

Comment:
The total number of lines is recommended to use getFileInfo

Reply:
Ok, good advice